app/bundles/UserBundle/Config/config.php

```diff
@@ -248,6 +248,10 @@
                     '%mautic.saml_idp_default_role%',
                 ],
             ],
+            'mautic.security.user_token_setter' => [
+                'class'     => \Mautic\UserBundle\Security\UserTokenSetter::class,
+                'arguments' => ['mautic.user.repository', 'security.token_storage'],
+            ],
```

Comment:
This can be removed due to autowiring
```suggestion
```

Reply:
@escopecz we have to do this because this class file is exist inside `Security` folder and security is exist in [exclude list](https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/DependencyInjection/MauticCoreExtension.php#L23)